flocker/control/httpapi.py

```diff
@@ -152,7 +152,10 @@ def _extract_containers_state(deployment_state):
     """
     result = []
     for node in deployment_state.nodes:
-        for application in (node.applications or ()):
+        applications = ()
+        if node.applications is not None:
+            applications = node.applications.values()
```

Comment:
Yea, turning this into: `((node.applications or {}).values())` just seemed too mean.